roles/usb_lib/files/usbmount/usbmount

```diff
@@ -96,22 +96,41 @@ if [ "$1" = add ]; then
     USAGE=$(echo "$DEVINFO"  | sed 's/.*[[:blank:]]USAGE="\([^"]*\)".*/\1/g; s/[[:blank:]]*//g;')
 
     if ! echo $USAGE | egrep -q "(filesystem|disklabel)"; then
-	log info "$DEVNAME does not contain a filesystem or disklabel"
+	log info "/$DEVNAME does not contain a filesystem or disklabel"
 	exit
     fi
 
-    # Try to use specifications in /etc/fstab first.
-    if egrep -q "^[[:blank:]]*$DEVNAME" /etc/fstab; then
-	log info "executing command: mount $DEVNAME"
-	mount $DEVNAME || log err "mount by DEVNAME with $DEVNAME wasn't successful; return code $?"
+    log debug "DEVNAME /$DEVNAME"
+    BOOTFW_DEV=$(/usr/bin/findmnt -no source /boot/firmware)
+    log debug "BOOTFW_DEV $BOOTFW_DEV"
+    if [ $BOOTFW_DEV = /$DEVNAME ]; then
+	log debug "/$DEVNAME contains filesystem type $FSTYPE"
+	log debug "skipping BOOTFS_DEV $BOOTFS_DEV mounted at /boot/firmware"
+	exit
+    fi
+    ROOT_DEV=$(/usr/bin/findmnt -no source /)
+    log debug "ROOT_DEV $ROOT_DEV"
+    if [ $ROOT_DEV = /$DEVNAME ]; then
+	log debug "/$DEVNAME contains filesystem type $FSTYPE"
+	log debug "skipping ROOT_DEV $ROOT_DEV mounted at /"
+	exit
+    fi
+    BOOT_DEV=$(/usr/bin/findmnt -no source /boot)
+    log debug "BOOT_DEV $BOOT_DEV"
+    if [ $BOOT_DEV = /$DEVNAME ]; then
+	log debug "skipping BOOTFS_DEV $BOOT_DEV mount as /boot"
```

Comment:
```suggestion
	log debug "/$DEVNAME contains filesystem type $FSTYPE"
	log debug "skipping BOOT_DEV $BOOT_DEV mount as /boot"
```

Reply:
1. First line of the 2 debug lines was missing.
2. Typo: BOOTFS_DEV should be BOOT_DEV.

ASIDE: Spaces generally much better than tabs, ideally yes if possible.